mods/example/rules/player.yaml

```diff
@@ -0,0 +1,10 @@
+^baseplayer:
+	AlwaysVisible:
+	Shroud:
+
+player:
+	Inherits: ^baseplayer
+	DeveloperMode:
```

Comment:
Maybe add `PlayerRadarTerrain` and `PlayerStatistics`?

Reply:
Base idea was to use common and minimal stuff to get you ingame but avoid crashes. Why do we need above mechanics loaded?

Reply:
Those two traits are important for proper functioning of the radar and observer UI respectively.